wagtailio/roadmap/models.py

```diff
@@ -74,7 +74,6 @@ def get_context(self, request, *args, **kwargs):
 class MilestoneItem(Orderable):
     NEEDS_SPONSORSHIP_LABEL = "needs sponsorship"
 
-    needs_sponsorship = models.BooleanField(default=False, editable=False)
     sponsorship_url = models.URLField(
```

Comment:
does this not need a migration?

Reply:
No it does not, see https://github.com/wagtail/wagtail.org/pull/477#issuecomment-2435942999

Reply:
ah, I missed your comment. Thank you